### PR TITLE
Commit yarn.lock, use >= for node, README fixes.

### PR DESCRIPTION
**Pull request by danvk**
Seems to work fine with node 9 as well as 8. The README mentions 6.

Also commits `yarn.lock`, which is standard practice to pin dependencies between developers.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "engines": {
-    "node": "8.9.1"
+    "node": ">=8.9.1"
```
Comment (patcon) [inline package.json]: Thanks! Not sure, but I think this is invalid, and will confuse our heroku deploys:
https://devcenter.heroku.com/articles/nodejs-support#specifying-a-node-js-version

It apparently accepts loose contraints, but not via operators

Comment (patcon) [reply package.json]: (I don't think this is the cause of the current deploy failure though :)

**Comment by patcon** [comment]
+1 to committing yarn.lock as best-practice

**Comment by patcon** [comment]
cc: @emarchak re: https://github.com/CivicTechTO/women-and-color-frontend/pull/20

File: package.json (hunk 1 above)
Comment (debchn) [reply package.json]: 👋the syntax is valid! (See example in https://docs.npmjs.com/files/package.json#engines) 👍

**Comment by patcon** [comment]
Don't feel right merging this until it gets review of @emarchak, since she was the one who removed the lockfiles last time. What are your thoughts on this, Marchak?

**Comment by patcon** [comment]
Oh, wait. `yarn.lock` is back on mainline :tada: 

I'm going to close this out, as it seems everything has been pulled in, but thanks so so much @danvk! I genuinely apologize that this PR fell between the cracks

**Comment by patcon** [comment]
Friendly point of order @s-kennedy and @emarchak: in future I would recommend we make sure that the changes of new contributors get prioritized above existing contributors. First-time contributors are REALLY important to an open source project, because how their changes are accepted (or not) affects their future contribution velocity

ie. this lockfile addition was, in open source land, much more valuable than an existing project contributor adding it :heart: 

Also, [using the `--author` flag](https://www.git-tower.com/learn/git/faq/change-author-name-email) allows easy fist-bumps to the folks who spiritually contributed to a change :)

Anyhow, just wanted to surface that before winding this PR down! Y'all are awesome

**Pull request closed without merging**